### ISSUE TITLE
Add Feedback and Contact Us section

### ISSUE THREAD

**Issue by Ashwinib26** [AlfiyaSiddique/TastyTrails#40]
To improve user engagement and allow visitors to easily provide feedback and get in touch, I suggest adding both a Feedback Section and a Contact Us Section to the website. This will create a direct line of communication with users and help gather valuable input about their experience with the food services.

Feedback Section: Add a user-friendly feedback form where customers can rate their experience, provide comments on food and service, and submit their feedback, with an engaging visuals aligned with the website's theme.

Contact Us Section: Include a simple contact form for users to reach out, along with essential contact details (phone, email, address) and as of now the social media links redirects to the project admin's contact which should be redirecting to website's social media links.

**Comment by INam1995** [comment]
please assign it to me @AlfiyaSiddique

**Comment by AlfiyaSiddique** [comment]
@INam1995 Issues are assigned on the first come first serve basis.
I am sure there are plenty of issues for everyone to contribute. Free feel to open new issues.

**Comment by AlfiyaSiddique** [comment]
@Ashwinib26 
Thanks for opening this issue.

Right now, we don't need a contact form.
And for the feedback form, its frontend is already being implemented by #11.
Are you interested in working on the backend part of it? If yes let me know and I will assisgn.

**Comment by Ashwinib26** [comment]
@AlfiyaSiddique 
Yes Sure , I can do the backend part of it , you can assign me the same.
Can you please provide some details that what exactly the functionalities you would be expecting there ?

**Comment by AlfiyaSiddique** [comment]
@Ashwinib26 When the no. of stars and message(optional) is sent from the frontend you have to store it in a model.
Now designing the model is up to you and just make it as efficient as you can. That's it just the feedback data should be stored in database. You can see the code and decide if a new collection(model) is needed to store feedback or it can be implemented in any of the already present collections.

**Comment by AlfiyaSiddique** [comment]
Let me know if you can do it? @Ashwinib26

**Comment by Ashwinib26** [comment]
@AlfiyaSiddique 
Thank you for the opportunity and for considering me for this task. I gave the backend implementation a try, but unfortunately, I wasn't able to achieve the desired results.
I would be glad to concentrate on areas where I can be more effective or collaborate with the team to address the issues.

**Comment by AlfiyaSiddique** [comment]
Okay, @Ashwinib26  no worries.
Thanks for showing interest

**Comment by AlfiyaSiddique** [comment]
Also @Ashwinib26, one more issue is there which you can  work on, it is as follows:
Currently, the feedback button is below on the footer and it appears for all states.
Can you make changes in frontend so that the the button on footer appears only  if the user is logged in and the name and email field of the feedback form is already filled in with the logged-in user's name and email and cannot be edited.
Let me know if you can do this.

**Comment by Ashwinib26** [comment]
Yes Sure @AlfiyaSiddique I can do this task , thank you for the opportunity.

**Comment by Ashwinib26** [comment]
Please assign me the issue @AlfiyaSiddique.

**Comment by AlfiyaSiddique** [comment]
Assigning to you @Ashwinib26
